indra/newview/llpanelpeople.cpp

```diff
@@ -894,13 +894,31 @@ void LLPanelPeople::updateNearbyList()
 // [/RLVa:KB]
         LLWorld::getInstance()->getAvatars(&mNearbyList->getIDs(), &positions, gAgent.getPositionGlobal(), ALControlCache::NearMeRange);
 
+        int count_in_region = 0;
         LLViewerRegion* cur_region = gAgent.getRegion();
-        mNearbyCountText->setTextArg("[COUNT]", std::to_string(mNearbyList->size()));
 
-        if (cur_region)
+         if (!cur_region)
+         {
+            LL_WARNS() << "Current region is null" << LL_ENDL;
+            return;
+        }
+
+        // Iterate through avatars in the region.
+        // The nearby list reports the avatars in 4096m range (ALControlCache::NearMeRange)
+        // Reported UUIDs may not be in same region.
+        // Also the TOTAL changes based on your filter results --FLN
+        for (size_t i = 0; i < positions.size(); ++i)
         {
             mNearbyCountText->setTextArg("[REGION]", cur_region->getName());
```

Comment:
This line should no longer be necessary since you're setting the `[REGION]` tag below outside of the loop.

Reply:
AhhhhhhH! LOL Thanks @Ruimuuv good catch. Old eyes missed that. Will re-submit a new PR for this 😫